src/Executor/Executor.php

```diff
@@ -26,9 +28,14 @@ class Executor
         'content-type' => '',
     ];
 
-    public function __construct(string $endpoint = 'http://appwrite-executor/v1')
-    { 
-        $this->endpoint = $endpoint;
+    public function __construct(string $endpoint = self::DEFAULT_HOST)
+    {
+        if (empty($endpoint)) {
+            Console::warning('Undefined Executor host (fallback to ' . self::DEFAULT_HOST . ')');
```

Comment:
No need to print anything to the log, and exception should do the trick and alert the maintainers.